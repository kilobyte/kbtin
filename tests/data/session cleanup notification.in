#ali down {#ali down #delay 1 #zap}
#hook close #main down
#run a sh -c 'sleep 2'
#run b sh -c 'echo foo'
