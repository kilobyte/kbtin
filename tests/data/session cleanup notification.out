foo

#SESSION 'b' DIED.
#SESSION 'a' ACTIVATED.

#SESSION 'a' DIED.
#THERE'S NO ACTIVE SESSION NOW.
